source/include/core_json.h

```diff
@@ -106,7 +109,6 @@ JSONStatus_t JSON_Validate( const char * buf,
  * @param[in] max  size of the buffer.
  * @param[in] queryKey  The key to search for.
```

Comment:
Should the parameter be renamed to `queryKeyOrIndex` now that this function supports both keys and array indices?

Reply:
I'll change it to be query, which is generic.